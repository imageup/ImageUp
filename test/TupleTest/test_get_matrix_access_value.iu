function main() -> int
{
	matrix : x(2, 2) = [1, 2| 3, 4];
	tuple : a = (x[0][0], x[1][1]);
	printf(a[0]);
	printf(a[1]);
	prints("correct output should be: 1, 4");
	return 0;
}