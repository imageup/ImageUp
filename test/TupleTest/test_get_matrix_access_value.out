1
4
correct output should be: 1, 4
